pennylane_qiskit/qiskit_device.py

```diff
@@ -82,29 +82,26 @@ def pauli_eigs(n):
     "RZ": ex.RZGate,
     "S": ex.SGate,
     "T": ex.TGate,
+
+    # Adding the following for conversion compatibility
+    "Sdg": ex.SdgGate,
+    "Tdg": ex.TdgGate,
     "CSWAP": ex.FredkinGate,
     "CRZ": ex.CrzGate,
     "PhaseShift": ex.U1Gate,
     "QubitStateVector": ex.Initialize,
-
-    # TODO: add the CRY gate, once the U3Gate is a part of PennyLane
-    # "CRY": U3Gate,
```

Comment:
Did you work out what this TODO meant?

Reply:
The confusing thing is that `CRY` is a one qubit gate, but `U3` is a 2 qubit gate...